tests/test_coverage.py

```diff
@@ -53,3 +53,30 @@ def test_coverage_symmetric_upper(request):
     assert (tot_cov == np.array([3, 1, 2])).all()
     assert clr.info["cis"] == 1
     assert clr.info["sum"] == 3
+    
+def test_coverage_weight(request):
```

Comment:
maybe `test_balanced_coverage` ? it is usually called `weight` but people usually refer to "balancing" and balanced-data - it's more readable